tests/Unit/Http/Form/SignupFormTest.php

```diff
@@ -401,7 +401,7 @@ public function dataGetsSanitizedCorrectly($inputData, $expectedData)
         $form = new \OpenCFP\Http\Form\SignupForm($inputData, $this->purifier);
         $form->sanitize();
         $sanitizedData = $form->getCleanData();
-        $this->assertEquals(
```

Comment:
While I am personally 👍 for being as strict as possible, the only limitations I see are

* when we really don't care that much about being that strict
* the output of `assertSame()` is not as helpful as that of `asserrEqual()` when comparing arrays

But if that's not an issue, totally 👍!

Reply:
If its a big issue we could also do something with [PHPUnitGoodPractices](https://github.com/PHPUnitGoodPractices/Traits) instead, which i think just reports that we are using Equal instead of Same. 

So we don't 'enforce' it strictly, but we are still notified if we don't.

Reply:
I'd be fine rolling forward and if it becomes a problem (array-diffs), we can always adjust. Tests should make our lives easier :)